build.gradle.kts

```diff
@@ -9,6 +11,32 @@ configurations {
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+  implementation("org.springframework.boot:spring-boot-starter-security")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+
+  implementation("org.flywaydb:flyway-core:8.4.3")
+  implementation("com.vladmihalcea:hibernate-types-52:2.14.0")
+  runtimeOnly("org.postgresql:postgresql")
+
+  implementation("org.springdoc:springdoc-openapi-ui:1.6.5")
+  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.5")
+  implementation("org.springdoc:springdoc-openapi-data-rest:1.6.5")
+
+  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.13.1")
+
+  implementation("com.zaxxer:HikariCP:5.0.1")
+  runtimeOnly("com.h2database:h2:2.1.210")
```

Comment:
there was a debate about this, and whether when we use Postgres we just use a docker image including test containers - I would normally on the side of docker, but I guess there data tables are going to be so simple it perhaps doesn't matter. I think the HMPPS audit api is an example of using Postgres for local/testing

Reply:
Ill merge this PR for now, and consider as a separate step